web/pgadmin/tools/sqleditor/utils/apply_explain_plan_wrapper.py

```diff
@@ -25,3 +25,17 @@ def apply_explain_plan_wrapper_if_needed(manager, sql):
         return render_template(template_path, sql=sql['sql'], **explain_plan)
     else:
         return sql['sql']
+
+
+def return_explain_query_length(query):
+    """
+    This method returns query length if it is explain query.
+
+    Args:
+        query: sql query
+    """
+    query = query.decode()
+    if query.startswith("EXPLAIN"):
```

Comment:
What will happen if someone types a `EXPLAIN` query with an incorrect table and executes the query?

Reply:
This line only checks if the given query is `EXPLAIN` query or not, if the table is incorrect it will highlight the table name. Attached screenshot
<img width="1575" alt="Screenshot 2023-12-07 at 1 01 15 PM" src="https://github.com/pgadmin-org/pgadmin4/assets/111979571/582284e8-f65c-428e-b88f-266a4c8fb458">

Reply:
I asked since you've checked if the query starts with `EXPLAIN` which is a possibility without using Explain button.